config/initializers/version.rb

```diff
@@ -0,0 +1,14 @@
+module GitRepo
+  def self.last_tag
+    begin
+      IO.popen("git describe --tags --abbrev=0 "){ |v| v.readline.strip }
```

Comment:
I think the `--abbrev=0` could be omitted so that people know if they are running the latest code of a branch instead of a release. But it's okay as is. Definitely an improvement.